incubator/hnc/e2e/tests/delete-anchor-crd_test.go

```diff
@@ -0,0 +1,35 @@
+package test
+
+import (
+	"os/exec"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Delete-anchor-crd", func() {
+	It("should create parent and deletable child", func() {
+		// create a namespace delete-crd-parent
+		cmd := exec.Command("kubectl", "create", "ns", "delete-crd-parent")
+		Expect(cmd.Run()).Should(BeNil())
```

Comment:
Feel free to start building utility functions if you like. Eg this could be something like:

`RunSuccessfully("kubectl create ns delete-crd-parent")`

where `RunSuccessfully` creates the command, runs it and validates that it succeeds.

Reply:
Or you could build something higher-level like `CreateNamespace("parent")`

Reply:
Added function runSuccess()